clickplc/driver.py

```diff
@@ -73,11 +83,17 @@ async def set(self, address, data):
 
         i = address.index(next(s for s in address if s.isdigit()))
         category, index = address[:i].lower(), int(address[i:])
-        if category not in self.supported:
-            raise ValueError("{} currently unsupported.".format(category))
+        if category not in self.data_types:
+            raise ValueError(f"{category} currently unsupported.")
+        data_type = self.data_types[category].rstrip(digits)
+        for datum in data:
+            if isinstance(datum, int) and data_type == 'float':
```

Comment:
Your comment on int/bool.

How are we merging this btw? Does this supersede Alex's PR?

Reply:
my branch got merged into numat/add_tags before this merge